sansa-query-spark/src/test/scala/net/sansa_stack/query/spark/dof/TestBase.scala

```diff
@@ -0,0 +1,41 @@
+package net.sansa_stack.query.spark.dof
+
+import net.sansa_stack.query.spark.dof.node.Helper
+import net.sansa_stack.rdf.spark.io._
+import org.apache.spark.sql.SparkSession
+import org.junit.runner.RunWith
+import org.scalatest.{ BeforeAndAfterAll, BeforeAndAfterAllConfigMap, ConfigMap, FlatSpec, Matchers }
+import org.scalatest.junit.JUnitRunner
+
+// @RunWith(classOf[JUnitRunner])
+class TestBase extends FlatSpec with Matchers with BeforeAndAfterAllConfigMap /* BeforeAndAfterAll */ {
```

Comment:
Not necessary to have it. We use a [spark-testing-base](https://github.com/holdenk/spark-testing-base) as a testing environment, so there is no need to have such tests.

Reply:
removed